bin/startNanoporePipeline.sh

```diff
@@ -360,9 +360,9 @@ else
 		testCode=$(tail -n 1 "${sampleSheet}" | awk -v sampleSheetFieldIndex="${sampleSheetFieldIndex}" 'BEGIN {FS=","}{print $sampleSheetFieldIndex}')
 
 		if [[ "${testCode}" == "LX001" ]]; then
-			bed_file="${TMP_ROOT_DIR}/software/nanopore/resources/LX001_v1.0.0.bed"
+			bed_file="${TMP_ROOT_DIR}/software/nanopore/resources/LX001_v1.0.1.bed"
 		elif [[ "${testCode}" == "LX002" ]]; then
-			bed_file="${TMP_ROOT_DIR}/software/nanopore/resources/LX002_v1.0.0.bed"
+			bed_file="${TMP_ROOT_DIR}/software/nanopore/resources/LX002_v1.0.1.bed"
```

Comment:
waarom zit dit de software van nanopore in de tmp_root_dir en niet in de apps/software?
En kunnen we niet iets bedenken met latest.bed (wat in werkelijkheid een symlink is naar LX002_v1.0.1.bed, scheelt je releases maken als je files als deze wil updaten